pkg/orderers/protobufs.go

```diff
@@ -1,14 +1,8 @@
 package orderers
```

Comment:
This whole file can be deleted after
1. The other comments are addressed (removing the need for `OrdererEvent`) and
2. the definition of `InstanceParams` is moved elsewhere (or even inlined).

Reply:
done (inlined for now, but this relates to how it is done for other modules and to try and make a skeleton consistent with them all. In the multisigcollect, this is part of the common package).